Tiltfile

```diff
@@ -103,7 +103,7 @@ local_resource(
 
 local_resource(
     "restic_binary",
-    cmd = 'cd ' + '.' + ';mkdir -p _tiltbuild/restic; BIN=velero GOOS=linux GOARCH=amd64 GOARM="" RESTIC_VERSION=0.13.1 OUTPUT_DIR=_tiltbuild/restic ./hack/build-restic.sh',
+    cmd = 'cd ' + '.' + ';rm -rf ../restic; BIN=velero GOOS=linux GOARCH=amd64 GOARM="" RESTIC_VERSION=0.13.1 OUTPUT_DIR=../velero/_tiltbuild/restic ./hack/build-restic.sh',
```

Comment:
I'm not sure we'd want to run `rm -rf` without confirmation.

If your $PWD is /git/velero and you have /git/restic as well wouldn't this command delete restic dir outside of velero repo?

Reply:
yes, you're right about the concern, I don't think `rm -f` should be in this file when it removes outside files. Maybe we need to change the restic download shell script and let all the thing under the Velero repo is good.

The origin reason is someone committed `./hack/build-restic.sh`, and don't notice the file was also used by tilt up, so broke the restic download behavior in `tilt up`.

I suggest creating a new hack script for `tilt` to download the restic file, although there would be some redundant code.

Reply:
could nest this under Makefile target `./path/to/restic/code:`
and then add this path to gitignore.